src/main/scala/beam/sim/BeamMobsim.scala

```diff
@@ -122,7 +122,7 @@ class BeamMobsim @Inject()(
           "RideHailManager"
         )
         context.watch(rideHailManager)
-        Await.result(rideHailManager ? Identify(0), timeout.duration)
+//        Await.result(rideHailManager ? Identify(0), timeout.duration)
```

Comment:
This is somewhat pertinent to https://github.com/LBNL-UCB-STI/beam/issues/1341 so I'm curious - are we sure that we do not need to wait for the RHM to be inited fully?

Reply:
@sfwatergit may comment on it.